src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs

```diff
@@ -12,7 +12,6 @@ public class IDispatchImplAttributeTests
         private const string ValueName = "Value";
 
         [Theory]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/50717", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
```

Comment:
If you rewrite the `Type type = typeof(HandleCollector).Assembly.GetType(TypeName);` line below (can't comment on that one directly) as `Type type = Type.GetType("System.Runtime.InteropServices.IDispatchImplAttribute, System.Runtime.InteropServices")` illink will be able to analyze this and the descriptor XML won't be needed.

Reply:
Why it works this way, but not the original way?

Reply:
It does seem like `typeof(XXX).Assembly.GetType(hard-coded string)` should be something the linker could recognize....

Reply:
The illinker doesn't have the necessary intrinsics to understand the pattern. It wasn't deemed to be common enough to be in a situation where we both know the assembly and the name.